ui/packages/consul-ui/translations/common/en-us.yaml

```diff
@@ -81,4 +81,6 @@ validations:
     error: "{name} must be a valid DNS hostname."
   immutable:
     help: Once created, this cannot be changed.
-
+  server-external-addresses:
+    help: |
+      Enter a fallback server address for this peer to be used in the event of failed automatic updates. This field is required for HCP-managed clusters.
```

Comment:
I wonder if something like "Enter a comma separated list of fallback server address(es) ..." might help user understand how to enter multiple?